### PR TITLE
made changes

### PR DESCRIPTION
**Pull request by TonyPNguyen**
Hey can you let me know if this is good

### CODE REVIEW

File: AtavusUserMatcher/ClientApp/src/pages/AdminLogin.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import AuthService from "../services/authService";
+
+const AdminLogin = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const navigate = useNavigate();
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    console.log(username);
```
Comment (AndresM125) [inline AtavusUserMatcher/ClientApp/src/pages/AdminLogin.js:12]: why logs in prod?

**Review by AndresM125 — CHANGES_REQUESTED** [review]

File: AtavusUserMatcher/ClientApp/src/App.js [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
-import { Layout } from './components/Layout';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AdminLogin from './pages/AdminLogin';
 import './custom.css';
+import Index from './pages/Index';
 
-export default class App extends Component {
-  static displayName = App.name;
-
-  render() {
```
Comment (AndresM125) [inline AtavusUserMatcher/ClientApp/src/App.js:10]: should be there

File: AtavusUserMatcher/ClientApp/src/pages/AdminLogin.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react"
+import { useNavigate } from "react-router-dom";
+import AuthService from "../services/authService";
+
+const AdminLogin = () => {
```
Comment (AndresM125) [inline AtavusUserMatcher/ClientApp/src/pages/AdminLogin.js:5]: hi

File: AtavusUserMatcher/ClientApp/src/pages/Index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+const Index = () => {
+
+  const [selectedFile, setSelectedFile] = useState(null);
+  const [selectedFileName, setSelectedFileName] = useState("No file selected");
+  const [data, setData] = useState("");
+  const inputFile = useRef();
+
+  const onSelectFileClicked = () => {
+    inputFile.current.click();
+  }
```
Comment (AndresM125) [inline AtavusUserMatcher/ClientApp/src/pages/Index.js:12]: test

File: AtavusUserMatcher/ClientApp/src/App.js (hunk 2 above)
Comment (TonyPNguyen) [reply AtavusUserMatcher/ClientApp/src/App.js:10]: lol....

**Pull request merged**